odl/operator/oputils.py

```diff
@@ -190,26 +195,15 @@ def power_method_opnorm(op, xstart=None, maxiter=100, rtol=1e-05, atol=1e-08,
         raise ValueError('`maxiter` must be positive, got {}'
                          ''.format(maxiter_in))
 
-    if op.domain == op.range:
```

Comment:
This whole PR could be changed to changing this line to `if op.adjoint is op:`, are we sure that's not a more sensible change?

Reply:
Oh I didn't even think of that :man_facepalming: 

Yes, that makes sense. I was kind of regretting the change of behavior already...
Let me change it.